### PR TITLE
Combining two features into one PR: homepage styling and messages styling

### PR DESCRIPTION
**Pull request by mike-zaharchenko17**
Changes:

- Added bootstrap styling to the messages that we display from base.html
- Added logic to template to ensure we show the right message style depending on the message tag
- Changed the homepage layout
- Added bootstrap styling to home.html
- Modified bootstrap.css to feature a custom mt-6 (margin-top-6) class to give us additional top padding on the topmost row

Combining new changes (homepage styling) and changes from PR #82

### CODE REVIEW

File: user_profile/views.py [hunk 1]
```diff
@@ -90,7 +93,14 @@ def edit_profile(request):
         )
         if profile_form.is_valid():
             profile_form.save()
```
Comment (kravisankaran) [inline user_profile/views.py:95]: How does this show up? On the page or is it a logger with success message

Comment (mike-zaharchenko17) [reply user_profile/views.py:95]: It shows up on the page. Django has a messages module, which, when called from a view function, queues up an alert style message that we can use to communicate with the user.  In base.html, we have some logic that will check if the messages data structure has anything in it. If it does, it'll iterate over the messages that are queued and display them at the top of the page.

image

Comment (kravisankaran) [reply user_profile/views.py:95]: Nice that looks good

**Review by kravisankaran — APPROVED** [review]
Looks to good to merge, we really need these feature!

**Pull request merged**